File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        my_set = set(nums)
        n=len(my_set)
        l1= sorted(my_set)
        print(my_set)
        for i in range(n):
            nums[i]=l1[i]
        return n
        

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int l =0;
        int r =1;
        for(int i=1; i<nums.size();i++){
            if(nums[l]==nums[i]) continue;
            else{
                nums[l+1]=nums[i];
                l++;
            }
        }
        return l+1;
    }
};